workflow/rules/activity_only_features.smk

```diff
@@ -19,18 +17,31 @@ rule activity_only_features:
 	script:
 		"../scripts/activity_only_features.R"
 
-rule gen_final_features:
-	# We really only do a file renaming here
-	# This step should be replaced by other modules if they wish to add more features
+# add external features (modify this rule in future pipelines- note, outputs should have name INPUT COL and will be renamed later)
+rule add_external_features:
 	input:
-		predictions_extended = os.path.join(RESULTS_DIR, "{biosample}", "ActivityOnly_features.tsv.gz"),
+		predictions_extended = os.path.join(RESULTS_DIR, "{biosample}", "ActivityOnly_features.tsv.gz")
 	output:
-		final_features = os.path.join(RESULTS_DIR, "{biosample}", "final_features.tsv.gz")
+		plus_external_features = os.path.join(RESULTS_DIR, "{biosample}", "ActivityOnly_external_features.tsv.gz")
```

Comment:
To make the filename more clear that it has all features instead of just external ones

```suggestion
		plus_external_features = os.path.join(RESULTS_DIR, "{biosample}", "ActivityOnly_plus_external_features.tsv.gz")
```